roles/common/tasks/main.yml

```diff
@@ -150,11 +150,28 @@
 
 # run this last so we only update if run was successful
 - name: drop an motd with ursula metadata
-  template: src=etc/update-motd.d/90-ursula-motd dest=/etc/update-motd.d/90-ursula-motd mode=0755
+  template:
+    src: etc/update-motd.d/90-ursula-motd
+    dest: /etc/update-motd.d/90-ursula-motd mode=0755
   when: os == 'ubuntu'
+  template:
```

Comment:
even if it is for the same task but different os? based on the os it is being put in a different location